File: jvm/src/main/kotlin/org/faziodev/saywhatstarter/types/messages/GoogleLinkOutSuggestionMessage.kt
package org.faziodev.saywhatstarter.types.messages

data class GoogleLinkOutSuggestionMessage(val linkOutSuggestion: GoogleLinkOutSuggestion): GooglePlatformMessage(){
    companion object {
        fun create(name: String, uri: String): GoogleLinkOutSuggestionMessage =
            GoogleLinkOutSuggestionMessage(GoogleLinkOutSuggestion(name, uri))
    }
}

File: csharp/SayWhatStarterWebhook/Models/Messages/GoogleLinkOutSuggestionMessage.cs
namespace SayWhatStarterWebhook.Models.Messages
{
    public class GoogleLinkOutSuggestionMessage: GooglePlatformMessage
    {
        public GoogleLinkOutSuggestion LinkOutSuggestion { get; set; }

        public static GoogleLinkOutSuggestionMessage Create(string name, string uri)
        {
            return new GoogleLinkOutSuggestionMessage
            {
                LinkOutSuggestion = new GoogleLinkOutSuggestion
                {
                    DestinationName = name,
                    Uri = uri
                }
            };
        }
    }
}